src/app/members/components/MemberList.tsx

```diff
@@ -93,7 +93,15 @@ export const MemberList = ({ members }: { members: Props }) => {
     }
   }, [success, toast]);
 
-  const groupedData = useGroupedData(data, "shift");
+  const allShifts = [Shift.ShiftA, Shift.ShiftB, Shift.ShiftC , Shift.ShiftD];
+
+  const groupedData = allShifts.map((shift) => {
+    const usersInShift = data.filter((user) => user.shift === shift);
+    return {
+      shift: shift,
```

Comment:
```suggestion
      shift,
```

In JS, when key and values are same, we can omit values.